projects/packages/my-jetpack/_inc/state/selectors.js

```diff
@@ -59,6 +59,11 @@ const purchasesSelectors = {
 	isRequestingPurchases: state => state.purchases?.isFetching || false,
 };
 
+const chatAvailabilitySelectors = {
+	getChatAvailability: state => state.chatAvailability?.items || {},
```

Comment:
Then here we could just:

```suggestion
	getChatAvailability: state => state.chatAvailability.isAvailable,
```

and use it as a bool.